change/@ni-nimble-tokens-09048a45-9290-4f9a-a5d8-940ddd6007a4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Refactor token build output. This removes some source files from the built package.",
```

Comment:
Specifically the four files that are now in `source/styledictionary/properties`. These are source, not build output so nobody should have been using them (other than our colors story, which I updated). I verified that SLE wasn't using them but it's still correct to call this `major`.